server/src/database/migrations/postgres/010-slack-users.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE slack_users (
+    id            VARCHAR(8) UNIQUE NOT NULL,
+    username      TEXT,
```

Comment:
varchar?

Reply:
what should be?

Reply:
Performance-wise varchar and text are the same in postgres

Reply:
Is the `id` here our internal ID or slack-generated?
If internal why not `CHAR(8)` ?

EDIT: Looks like it's generated by `uuid(8)` so `CHAR(8)` should suffice